python/GafferUI/UIEditor.py

```diff
@@ -572,6 +581,55 @@ def __setValue( self, unused, value ) :
 
 		self._updateFromWidget( value )
 
+class _FileSystemPathMetadataWidget( _MetadataWidget ) :
+
+	def __init__( self, key, target = None, acceptEmptyString = True, **kw ) :
+
+		self.__row = GafferUI.ListContainer( GafferUI.ListContainer.Orientation.Horizontal, spacing = 4 )
+		self.__textWidget = GafferUI.TextWidget()
```

Comment:
Can we use `PathWidget` here instead, so we get the standard tab complete etc? PathPlugValueWidget should hopefully provide a decent example.

Reply:
How do you like the code in c7f3ff4? I've squashed in the changes as the diff is still pretty small.